### ISSUE TITLE
Feature request: use default camera icon

### ISSUE THREAD

**Issue by MikeBeas** [kb1ooo/snappy-public#21]
I like to keep Snappy in my dock and would like the option to have it link to the default camera app icon (like I do with biteSMS and the regular Messages icon). Not super important since I can manually drop in the icon myself but it might be nice to have.

**Comment by MikeBeas** [comment]
(Let me rephrase that: just have a symlink to the camera icon, not make it so that tapping the actual Camera app brings up snappy)

**Comment by AltF5** [comment]
This may be a good graphical option, but probably should be marked as low priority.
